File: web3rpc/rpc-specs/code-samples/python/eth/others/createAccessList.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

transactionArgs = {
    "from": "0x3bc5885c2941c5cda454bdb4a8c88aa7f248e312",
    "data": "0x20965255",
    "gasPrice": "0x3b9aca00",
    "gas": "0x3d0900",
    "to": "0x00f5f5f3a25f142fafd0af24a754fafa340f32c7"
}

w3 = Web3(Web3.HTTPProvider(host))
eth_response = w3.eth.create_access_list(transactionArgs)

print(eth_response)

File: web3rpc/rpc-specs/code-samples/javascript/eth/others/createAccessList.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");

    const blockNumberOrHash = 'latest'
    provider.eth.createAccessList(transactionArgs, blockNumberOrHash, {}, (err, data, response) => {
        console.log(data);
    });

}
)()